include/cantera/thermo/BinarySolutionTabulatedThermo.h

```diff
@@ -0,0 +1,191 @@
+/**
+ * @file BinarySolutionTabulatedThermo.h
+ * Header file for an binary solution model with tabulated standard state
+ * thermodynamic data (see \ref thermoprops and class
+ * \link Cantera::BinarySolutionTabulatedThermo BinarySolutionTabulatedThermo\endlink).
+ */
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at http://www.cantera.org/license.txt for license and copyright information.
```

Comment:
I think in new files we should link to `https://cantera.org/license.txt`, and as we edit older headers, update them to point to the new link.

Reply:
Fixed (for this class).